crates/bin/pclientd/Cargo.toml

```diff
@@ -64,6 +65,10 @@ assert_cmd = "2.0"
 base64 = "0.20"
 ibc-types = { version = "0.7.0" }
 ibc-proto = { version = "0.37.0", default-features = false, features = ["server"] }
+penumbra-proof-params = { path = "../../crypto/proof-params", features = [
+    "bundled-proving-keys",
+    "download-proving-keys",
+] }
```

Comment:
i think we don't want to unconditionally enable these features, if we have feature control of our own above. either we should enable them unconditionally and not have the features in this binary (seems preferable?) or we should only enable them conditionally.

Reply:
ditto missed that this was a dev feature